package.xml

```diff
@@ -18,6 +18,7 @@
   <build_depend>genmsg</build_depend>
 
   <run_depend>genmsg</run_depend>
+  <run_depend>python-yaml</run_depend>
 
   <test_depend>python-yaml</test_depend>
```

Comment:
With the added run dependency this test dependency need to be removed, see current CI failures:

> The test dependency on "python-yaml" is redundant with: run_depend